content/en/docs/Examples/localai.md

```diff
@@ -0,0 +1,74 @@
+---
+title: "LocalAI"
+linkTitle: "LocalAI"
+weight: 4
+description: > 
+    This section describe examples on how to deploy Kairos with k3s and LocalAI
+---
+
+{{% alert title="Warning" %}}
+This tutorial will download an AI model which is around 4Gib
+{{% /alert %}}
+
+Welcome to the guide on using LocalAI with Kairos and K3s on your nodes!
+
+But first, what is [LocalAI](https://github.com/go-skynet/LocalAI)?
+
+LocalAI is a self-hosted, community-driven simple local OpenAI-compatible API written in go. Can be used as a drop-in replacement for OpenAI, running on CPU with consumer-grade hardware. Supports ggml compatible models, for instance: LLaMA, alpaca, gpt4all, vicuna, koala, gpt4all-j, cerebras.
+
+This means that you can have the power of an AI model in your Edge-Kubernetes cluster, and it can all be easily done thanks to Kairos!
```

Comment:
I would also mention that we use gpt4all models, and make it clear the component sitting behind the scene here:
- LocalAI
- GPT4ALL
- Kairos (Kubernetes)